### PR TITLE
4.13.1.0 documentation release update

### PR DESCRIPTION
**Pull request by andrijapanicsb**
- List of fixed issues / GitHub PRs from from 4.13.0.0 to 4.13.1.0 (changes.rst)
- Upgrade instructions FROM 4.13.x (upgrade-4.13.rst)
- About page

### CODE REVIEW

**Comment by andrijapanicsb** [comment]
requesting docbuild

**Comment by andrijapanicsb** [comment]
requesting docbuild

**Comment by andrijapanicsb** [comment]
requesting docbuild

**Comment by andrijapanicsb** [comment]
requesting docbuild

File: source/releasenotes/changes.rst [hunk 1]
```diff
@@ -13,14 +13,542 @@
...
    under the License.
 
+Issues Fixed in 4.13.1.0
+========================
 
 
-Changes in |release| since 4.12
-===============================
+Issues Fixed in 4.13.1.0
+------------------------
```
Comment (DaanHoogland) [inline source/releasenotes/changes.rst:21]: why the double header?

File: source/releasenotes/changes.rst [hunk 2]
```diff
@@ -13,14 +13,542 @@
...
+.. _`#3636`: https://github.com/apache/cloudstack/pull/3636
+.. _`#3678`: https://github.com/apache/cloudstack/pull/3678
+.. _`#3605`: https://github.com/apache/cloudstack/pull/3605
+.. _`#3612`: https://github.com/apache/cloudstack/pull/3612
+.. _`#3666`: https://github.com/apache/cloudstack/pull/3666
+.. _`#3620`: https://github.com/apache/cloudstack/pull/3620
+.. _`#3648`: https://github.com/apache/cloudstack/pull/3648
+.. _`#3627`: https://github.com/apache/cloudstack/pull/3627
+.. _`#3608`: https://github.com/apache/cloudstack/pull/3608
+.. _`#3574`: https://github.com/apache/cloudstack/pull/3574
```
Comment (DaanHoogland) [inline source/releasenotes/changes.rst:547]: these links are already in the table above, why do we need them again?

File: source/releasenotes/changes.rst (hunk 1 above)
Comment (andrijapanicsb) [reply source/releasenotes/changes.rst:21]: Due yo headers in the ToC table on yhe left. Was weird for me as well, but needed as such.

File: source/releasenotes/changes.rst (hunk 2 above)
Comment (andrijapanicsb) [reply source/releasenotes/changes.rst:547]: Nope, not redundant, those are links for the table above. Some weird stuff, dont show when you actually see the docs that ape has built!

**Comment by alexandremattioli** [comment]
LGTM

**Comment by andrijapanicsb** [comment]
requesting docbuild

**Comment by andrijapanicsb** [comment]
requesting docbuild

**Comment by andrijapanicsb** [comment]
1 x LGTM, merging.

**Pull request merged**